src/calculateRentalCost.js

```diff
@@ -28,7 +28,25 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const PRICE_PER_DAY = 40;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+  const SHORT_TERM = 3;
+  const SHORT_TERM_DISCOUNT = 20;
+
+  let basePrice = days * PRICE_PER_DAY;
+
+  if (days >= LONG_TERM) {
+    const basePrice = days * PRICE_PER_DAY;
```

Comment:
You already declare variable with `basePrice` name.  
```suggestion
  const basePrice = days * PRICE_PER_DAY;

  if (days >= LONG_TERM) {
```